package.json

```diff
@@ -4,7 +4,7 @@
 	"private": true,
 	"description": "GitHub Actions used by VS Code for triaging issues",
 	"scripts": {
-		"test": "jest test --silent",
+		"test": "jest --silent",
```

Comment:
small fix, makes running jest properly easier.

Now you can run single file tests for example:

`yarn test -- writeDocsFiles.test.ts`